src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+because the page is so new that it hasn't been picked up yet or because the page has insufficient
+traffic to be included in the dataset. In cases like these, we attempt to fall back to more
+granular origin-level data whenever possible. This data represents the aggregate user experiences
+on all pages of the website, so while it may not be immediately relevant to your particular page
+experience, it should still offer some insight into how users experience the site as a whole.
+Another common case is to see a "Waiting for FID" message. This is because FID is the only
+Core Web Vital that requires user interaction to be measured, so we won't have any local FID data
+to show until you interact with the page.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/qt6fmKIjPNrUsckKiUfS.png", alt="Screenshot of the
+Web Vitals extension showing only local data; field data is unavailable.",
+width="800", height="548" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing only local data; field data is unavailable.
+  </figcaption>
+</figure>
+
+There are some less common edge cases that you may run into while using the extension. We talked
+about how some pages might have insufficient data, but this can also happen to entire origins.
+When that happens, the extension will behave just like it did before we added any field data, and
+it will just show you how your local experiences performed. How you interact with the page may also
+affect the relevance of the data. For example, loading the page in a background tab will count the
+entire time until you foreground the page against the LCP metric. In other words, if it takes you
+90 seconds to switch to the tab, the LCP might appear as 91.5 seconds. When this happens, a little
+warning icon will appear next to your LCP value to warn you of the artificial inflation. Note that
+these kinds of irrelevant LCP values are ignored in the CrUX dataset. One last edge case to be
+aware of is specific to FID. Because it depends on user interactions, as mentioned, that makes it
+less likely to be available in the dataset for pages/websites with fewer page views. So if you're
+visiting a page like that, you may only see field data for LCP and CLS. It's worth noting for the
+sake of completeness that the same could happen for LCP or CLS in extremely rare situations, but
+this issue is predominantly affected by FID.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/0K0g0TD22jdNZpSe3s3p.png", alt="Screenshot of the
+Web Vitals extension showing local metrics compared to phone field data.",
+width="800", height="556" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing local metrics compared to phone field data.
+  </figcaption>
+</figure>
+
+By default, all of the data in the Web Vitals extension corresponds to real desktop users'
```

Comment:
Also might be a good idea to mention the desktop thing somewhere earlier in the post.

Reply:
No need to edit, I missed it. I'm gonna break up that paragraph too so it's easier to parse.